onboard/merge_iterator_test.go

```diff
@@ -0,0 +1,91 @@
+package onboard
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/treeverse/lakefs/catalog/rocks/testutils"
+
+	"github.com/treeverse/lakefs/catalog/rocks"
+)
+
+func TestMergeIterator(t *testing.T) {
+	cases := []struct {
```

Comment:
I would be much happier if some test here had the same path in both `InvIt` _and_ `CommittedIt`, and it was part of the processed prefixes.

Reply:
You got it.